src/components/MultipleAvatars.js

```diff
@@ -74,6 +74,7 @@ const defaultProps = {
 function MultipleAvatars(props) {
     const [avatarRows, setAvatarRows] = useState([props.icons]);
     let avatarContainerStyles = props.size === CONST.AVATAR_SIZE.SMALL ? [styles.emptyAvatarSmall, styles.emptyAvatarMarginSmall] : [styles.emptyAvatar, styles.emptyAvatarMargin];
+    avatarContainerStyles = props.size === CONST.AVATAR_SIZE.SMALLER ? [styles.emptyAvatarSmaller, styles.emptyAvatarMargin] : avatarContainerStyles;
```

Comment:
Can we merge those?

Reply:
If merge with two ternary expressions, lint error "Do not nest ternary expressions." is shown. What do you think about this?

Reply:
Hmm, maybe write a function that accepts a size and in that function use `switch`.

Reply:
@s77rt Please recheck again.